src/backend/gunicorn.conf.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+bind = ":8085"
+accesslog = "-"
+errorlog = "-"
+workers = 3
+loglevel = "Debug"
```

Comment:
I think debug level logging makes a lot of sense for local dev. What is our usual practice for staging and production? Should it be dependent on the env if they differ?

Reply:
We can certainly vary this - and now programmatically based on environment.  But AFAICT all Gunicorn does for debug is spit out all the values it's been set with. ANd this is just debug for gunicorn not Django.

Reply:
Got it. If this is the case, then I think keeping this as is seems fine 👍🏼